.vsts-ci.yml

```
@@ -62,6 +55,21 @@ variables:
       /p:DotNetRuntimeSourceFeedKey=$(dotnetbuilds-internal-container-read-token-base64)
       /p:dotnetbuilds-internal-container-read-token-base64=$(dotnetbuilds-internal-container-read-token-base64)
 
+- ${{ if and(eq(variables['System.TeamProject'], 'public'), eq(variables['Build.Reason'], 'PullRequest')) }}:
+  - name: VmrRef
+    value: $(System.PullRequest.TargetBranch)
+- ${{ else }}:
+  - name: VmrRef
+    value: 'refs/heads/main'
```

Btw this also works in YAML:
```suggestion
    value: refs/heads/main
```